packages/mobile/ios/AudiusReactNative/Info.plist

```diff
@@ -1,106 +1,110 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleDisplayName</key>
+	<string>${APP_DISPLAY_NAME}</string>
```

Comment:
Mind giving me the quick info about changes for this file?

Reply:
xcode auto formatted most of it. i just added co.audius to the universal link section :(